### DISCUSSION TITLE
Show only files in the media library associated to the module.

### DISCUSSION THREAD

**Discussion by jpmateo022** [area17/twill#1250] [Q&A]
Hello,

I'm new to twill and I would like to ask a question. Is it possible to only show files/images in the media library that are only associated with that module? Because whenever I upload all images/files are being displayed including making it harder to select the files that have been uploaded.

**Comment by pboivin** [comment]
Hi @jpmateo022, I don't think this will be possible out of the box, because no module information is attached to individual images when they are uploaded to the Media Library. Images are associated to specific records only when they are selected in a `medias` field.
